lib/reactotron-core-server/src/reactotron-core-server.ts

```diff
@@ -13,8 +13,7 @@ import {
   WssServerOptions,
 } from "./types"
 import { readFileSync } from "fs"
-import type { MittStatic } from "mitt"
-const Mitt: MittStatic = require("mitt") // Jest breaks with esm import, use cjs here ¯\_(ツ)_/¯
```

Comment:
Jest is not happy about handling for this for some reason :( https://app.circleci.com/pipelines/github/infinitered/reactotron/1388/workflows/753cdc32-7102-4549-8bc1-f65dd1de26c0/jobs/1565